pkg/addonmanager/controllers/agentdeploy/utils.go

```diff
@@ -518,18 +517,17 @@ func getManifestConfigOption(agentAddon agent.AgentAddon,
 		}
 	}
 
-	emc := agentAddon.GetAgentAddonOptions().ManifestConfigs
-	for _, mc := range emc {
+	aaomc := agentAddon.GetAgentAddonOptions().ManifestConfigs
```

Comment:
what does aaomc stands for :). You might want a more meaningful name, event it is longer.

Reply:
agentAddonOptionsManifestConfigs, but I changed it to userConfiguredManifestConfigs